packages/core/src/sdk/ui/useScreenResize.ts

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react'
+
+const MAX_MOBILE_WIDTH = 420
```

Comment:
suggestion: Adding the reason of this number.

```suggestion
// We are using the Moto G Power device measurement as a reference, as used by PageSpeed Insights.
const MAX_MOBILE_WIDTH = 420
```